app/src/main/java/com/example/triptracker/viewmodel/MapViewModel.kt

```diff
@@ -27,12 +29,17 @@ import kotlinx.coroutines.launch
  */
 class MapViewModel(
     val geocoder: NominatimApi = NominatimApi(),
-    private val pathList: MutableLiveData<ItineraryList> = MutableLiveData<ItineraryList>(),
+    val pathList: MutableLiveData<ItineraryList> = MutableLiveData<ItineraryList>(),
     private val repository: ItineraryRepository = ItineraryRepository(),
     val filteredPathList: MutableLiveData<Map<Itinerary, List<LatLng>>> =
         MutableLiveData<Map<Itinerary, List<LatLng>>>()
 ) : ViewModel() {
 
+  val DUMMY_SELECTED_POLYLINE =
+      SelectedPolyline(
```

Comment:
Could you please add a comment here on why this dummy_selected_polyline is necessary?
Thanks

Reply:
This is to avoid nullability issues ! I added a comment thanks !